doc/server-config.md

```diff
@@ -102,6 +102,8 @@ endpoint instead of AWS S3, such as `my-minio-server.example.com`. A specific
 region may be specified by using [Low-level Configuration](#lowlevel), though
 the bucket's region will generally be detected automatically.
 
+The `s3` storage option can be used for `b2` since it's possible to use AWS go SDK [backblaze article](https://help.backblaze.com/hc/en-us/articles/360047629713-Using-the-AWS-Go-SDK-with-B2). However since it requires the region to be provided, using [Low-level Configuration](#lowlevel) is preferred.
```

Comment:
Can't we find a way to include the region in the s3 config string instead?

Reply:
yes, I was asking myself the same thing..

Reply:
something like `key:secret:bucket[/optional/dir]:region` would be backwards-compatible

Reply:
Actually, region is not required when calling B2 API... It is possible to use b2 with the following block in the classical configuration file (ie: not in lowlevel):
```
[...]
"s3": "key:secret:hostname",
[...]
``` 
hostname has to be the endpoint displayed on backblaze interface when listing Buckets.
I have to cancel some change..